Adds ignore pattern and stdlib filter settings. (#415)
microsoft/vscode-pylint: README.md

Note:

-   This extension is supported for all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the `python` language (i.e., python >= 3.8).

> -   Shipped `pylint` version is used by default. You can use `pylint` from your environment by setting `pylint.importStrategy` to `fromEnvironment`. You can use a custom `pylint` executable by setting `pylint.path`.

Author: @luabud @cwebster-99 Can you provide suggestions for this line.

> -   Shipped `pylint` version is used by default. You can use `pylint` from your environment by setting `pylint.importStrategy` to `fromEnvironment`. You can use a custom `pylint` executable by setting `pylint.path`.

Reviewer: ```suggestion
-   By default, this extension uses the shipped `pylint` version. However, you can use `pylint` from your environment by setting `pylint.importStrategy` to `fromEnvironment`. Alternatively, you can use a custom `pylint` executable by setting `pylint.path`.
```

| pylint.lintOnChange     | `false`                                                                                                                                | (experimental) Setting to control linting on change feature.                                                                                                                                                                                                                                                             |
| pylint.ignorePatterns   | `[]`                                                                                                                                   | Glob patterns to exclude files and directories while linting.                                                                                                                                                                                                                                                            |

> | pylint.includeStdLib    | `false`                                                                                                                                | Setting to control whether to include standard library paths while linting.                                                                                                                                                                                                                                              |

Reviewer 2: ```suggestion
| pylint.includeStdLib    | `false`                                                                                                                                | Controls whether to perform linting on Python's standard library files or directories.                                                                                                                                                                                                                                        |
```

Reviewer 2: I'm wondering if we should add a note about how this relates to ignorePatterns ? E.g. if someone adds some stdlib file to ignorePatterns   but this  includeStdLib   is set to true, which one takes precedence? 
(we could add this scenario to our docs only instead of the setting description too)

Author: It should likely go in the docs. StdLib check occurs before pattern check.
